com.officefactoring.officefactoring.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.officefactoring.officefactoring
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+rename-icon: officefactoring
+command: OfficeFactoring
+modules:
+  - name: OfficeFactoring
+    buildsystem: simple
+    build-commands:
+      - install -D OfficeFactoring /app/bin/OfficeFactoring
+      - install -D libmidas.so.2 /app/bin/libmidas.so.2
+      - >-
+        install -D officefactoring.png
+        /app/share/icons/hicolor/256x256/apps/officefactoring.png
+      - >-
+        install -D com.officefactoring.officefactoring.metainfo.xml
+        /app/share/metainfo/com.officefactoring.officefactoring.metainfo.xml
+      - >-
+        install -D com.officefactoring.officefactoring.desktop
+        /app/share/applications/com.officefactoring.officefactoring.desktop
+    sources:
+      - type: archive
+        url: 'http://officefactoring.serveftp.com/demo/officefactoring-1.tar.gz'
+        sha256: 2404f77ce3015e710179aed1eef092c1c349f2751d0be1192ceeaa4abd6b1065
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /share/aclocal
+  - /man
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+  - /lib/girepository-1.0
+  - /share/doc
+  - /share/gir-1.0
+finish-args:
+  - '--socket=x11'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=wayland'
```

Comment:
if it support wayland then it's `fallback-x11` otherwise remove the wayland socket.